<launch>
  <group ns="urg2">
    <node pkg="urg_node" name="urg_node" type="urg_node">
      <param name="ip_address" value="192.168.0.10"/>
      <param name="angle_min" value="-1.832595714595"/>
      <param name="angle_max" value=" 1.832595714595"/>
    </node>
    <node pkg="tms_ss_pot" type="change_nan" name="change_nan2">
      <remap from="/most_intense" to="/urg2/most_intense"/>
      <remap from="/LaserTracker" to="/LaserTracker2"/>
    </node>
  </group>
</launch>
